spring-cloud-open-service-broker-contract-tests/src/test/resources/contracts/binding/reactive/async_in_progress_response.json

```diff
@@ -0,0 +1,3 @@
+{
+  "operation": "still working"
```

Comment:
Example be more realistic with sample used in the specs such as `task_10`

Reply:
sure, that's reasonable. thanks.